### ISSUE TITLE
Validation when setting a grouped RadioButton throws a RuntimeError: path to 'V' has indirects

### ISSUE THREAD

**Issue by leclerce** [pymupdf/PyMuPDF#2537]
## Describe the bug
See discussion #2534

## To Reproduce

Set a radio button in a group to "on":
```python
...
# Ex. Using a data dictionary with the field names as dict keys
if widget.field_type == fitz.PDF_WIDGET_TYPE_RADIOBUTTON:
     if my_data.get(widget.field_name) in widget.button_states()['normal']:
          widget.field_value = True 
          widget.update()
...
```

## Your configuration 

python 3.11.4 (main, Jun 14 2023, 18:25:14) [GCC 12.2.0] 
Linux  
PyMuPDF 1.22.3: Python bindings for the MuPDF 1.22.0 library.
Version date: 2023-05-10 00:00:01.
Built for Python 3.11 on linux (64-bit).

Wheel install

**Comment by JorjMcKie** [comment]
Thank you!
As discussed, this instruction is unnecessary anyway and will be removed for the next version.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.23.0.
